sktm/jenkins.py

```diff
@@ -320,6 +326,56 @@ def build(self, baserepo=None, ref=None, baseconfig=None,
         logging.info("submitted build: %s", build)
         return build.get_number()
 
+    def __get_job(self):
+        """
+        Get Jenkens job by job name. Retry Jenkins N times in case of temporary
+        network failures.
+
+        Return:
+            (job, None) if succeed, else return (None, e) where e is the last
+            exception.
+        """
+        retry_cnt = self.retry_cnt
```

Comment:
Is this really necessary? Why not use `self.retry_count` directly?

Reply:
Use self.retry_cnt instead.